sirepo/auth_db.py

```diff
@@ -140,6 +140,17 @@ def delete(self):
                 self._session().delete(self)
                 self._session().commit()
 
+        @classmethod
+        def delete_user(cls, uid):
+            """Delete user from all tables"""
+            for t in [x for x in cls.metadata.tables.values() if 'uid' in x.columns]:
+                cls._session().execute(
+                    sqlalchemy.delete(t).where(uid==uid).execution_options(
+                        synchronize_session='fetch',
```

Comment:
this should be encapsulated. I think there should be a context manager.

Reply:
efb9eef49 . I'm not sure how to do this in a contextmanager. I looked to set `syncrhonize_session` when creating the session but it can't be set then...